src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterParam } from '../../types/FilterParam';
+import { Todo } from '../../types/Todo';
+import {
+  checkCompletedTodo,
+  counterOfActiveTodos,
+  filterTodos,
+} from '../../helpers/helpers';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterParam,
+  setFilterType: (type: FilterParam) => void,
+  removeTodo: (todoId: number) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  setFilterType,
+  removeTodo,
+}) => {
+  const handleFilterChange = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const href = event.currentTarget.text;
+
+    switch (href) {
+      case FilterParam.All:
+        return setFilterType(FilterParam.All);
+
+      case FilterParam.Active:
+        return setFilterType(FilterParam.Active);
+
+      case FilterParam.Completed:
+        return setFilterType(FilterParam.Completed);
+
+      default:
+        return setFilterType(FilterParam.All);
```

Comment:
Try to rename `href` to a more meaningful value
Can't you just do smth like this:
```
const href = event.currentTarget.text || FilterParam.All;
setFilterType(href)